assets/scss/_card.scss

```diff
@@ -0,0 +1,97 @@
+.card-list {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+
+  @media all and (max-width: 768px) {
+    flex-direction: column;
+  }
+
+  li {
+    flex: 0 0 45%;
+    margin-bottom: 6rem;
+  }
+
+  li:nth-child(odd):last-child { margin-bottom: 0; }
+
+  .card-small {
+    padding: 2rem;
+
+    background-color: $bg-color;
+    border: 1px solid $alt-bg-color;
+    border-radius: 1.5rem;
+  }
+
+  .card {
+    display: inline-flex;
+  
+    flex-direction: column;
+    height: 100%;
+    width: 100%;
+    
+    background-color: $bg-color;
+    border: 1px solid $alt-bg-color;
+    border-radius: 1.5rem;
```

Comment:
I have to say that I didn't like the "look" of the cards. It seems that they don't fit well with the rest of the theme.
I gonna try some modifications and come back with ideas later.

Reply:
![Screenshot from 2020-05-20 20-56-49](https://user-images.githubusercontent.com/1427376/82508828-9ca89d00-9adc-11ea-9bfd-2cdbacd476c1.png)

Cards with border-radius of `0.5` look way better.
I'm happy with these font sizes. I gonna get some design advice.

Reply:
But, I don't know. Cards look very strange without shadows.
Maybe we should look for another listing option for projects.

Reply:
I was thinking if the project list would look better using post listing style like these:

![post-list](https://user-images.githubusercontent.com/1427376/82679853-dfc05880-9c21-11ea-9b61-8daed8ee03fd.png)
![Show_posts_EN-768x368](https://user-images.githubusercontent.com/1427376/82679858-e0f18580-9c21-11ea-94ed-203e956c0ebe.png)

Reply:
I think that projects should be displayed one line at a time, like posts but with more informations. It will use less space in height.
Do you want to display them in cards or not ?

Reply:
Have you pushed these changes? I need to test some ideas.

Reply:
No, I did that quickly in the browser.